app/models/household_good.rb

```diff
@@ -1,9 +1,11 @@
 class HouseholdGood < ApplicationRecord
-  validates :weight, presence: true
-  validates :name, presence: true, uniqueness: { scope: :household_good_category_id }
-
   belongs_to :household_good_category
+
   validates :household_good_category, presence: true
+  validates :name, presence: true, uniqueness: { scope: :household_good_category_id }
+  validates :weight, presence: true
+
+  default_scope { order(name: :asc) }
```

Comment:
The [`default_scope` method](http://api.rubyonrails.org/classes/ActiveRecord/Scoping/Default/ClassMethods.html#method-i-default_scope) basically does what its name implies:

> Use this macro in your model to set a default scope for all operations on the model.